app/views/application/personalDetails.scala.html

```diff
@@ -133,44 +134,33 @@ <h3 class="heading-medium">Qualification requirements</h3>
                   <tr>
                     <td>
                       <ul class="">
-                        @availableSchemes.filter(scheme => !scheme.requiresALevel && !scheme.requiresALevelInStem).map { scheme =>
-                          <li>@scheme.name</li>
-                        }
+                        <li>Business</li>
+                        <li>Commercial</li>
+                        <li>Finance</li>
                       </ul>
                     </td>
-                    <td>At least 5 GCSEs, grades C or above including English language
-                      and maths, <a href="https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels" rel="external" target="_blank">or equivalent</a>.
-                      <br>
-                      <p>From 2017, the equivalent GCSE grades are 4 or above.</p>
+                    <td>
+                      At least 5 GCSEs at grade C and above, or the new grade 4 and above, including English Language and Maths, or <a href="https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels" rel="external" target="_blank">equivalent</a>.
                     </td>
                   </tr>
                   <tr>
                     <td>
                       <ul class="">
-                        @availableSchemes.filter(scheme => scheme.requiresALevel && !scheme.requiresALevelInStem).map { scheme =>
-                          <li>@scheme.name</li>
-                        }
+                        <li>Project delivery</li>
```

Comment:
Ohhh good spot! That's a copy and paste error. The scheme names can be reverted to the dynamic listings, as long as the qualification requirements are left updated.